CMakeLists.txt

```diff
@@ -28,11 +31,15 @@ set(CMAKE_CXX_STANDARD 11)
 set(RADIO_DIRECTORY ${PROJECT_SOURCE_DIR}/radio)
 set(RADIO_SRC_DIRECTORY ${RADIO_DIRECTORY}/src)
 set(COMPANION_SRC_DIRECTORY ${PROJECT_SOURCE_DIR}/companion/src)
+set(SIMU_SRC_DIRECTORY ${COMPANION_SRC_DIRECTORY}/simulation)
 
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake)
 
+# options shared by all targets
+option(VERBOSE_CMAKELISTS "Show extra information while processing CMakeLists.txt files." OFF)
+option(WARNINGS_AS_ERRORS "Treat all compiler warnings as error." OFF)
```

Comment:
That would probably be better.  Or `Treat any compiler warning as an error.`  It only takes one  :)